actions/event_calendar/addtocalendar.php

```diff
@@ -16,12 +16,16 @@
 		
 	if (event_calendar_add_personal_event($event_guid, $user_guid)) {
 		remove_entity_relationship($user_guid, 'event_calendar_request', $event_guid);
-		notify_user($user_guid, elgg_get_site_entity()->guid, elgg_echo('event_calendar:add_users_notify:subject'),
-			elgg_echo('event_calendar:add_users_notify:body', array(
-			$user->name,
-			$event->title,
-			$event->getURL()
-		)));
+		if ($user_guid != elgg_get_logged_in_user_guid()) {
+			$user_language = ($user->language) ? $user->language : (($site_language = elgg_get_config('language')) ? $site_language : 'en');
```

Comment:
Ugh, it seems that in case of recipient language missing, elgg_echo() will default to the language of the logged in user instead of the site language. This is really a bug of elgg_echo(), but too late to it fix for Elgg 1.9 now.